ios/Runner/AppAuthHandler.swift

```diff
@@ -117,20 +143,27 @@ class AppAuthHandler {
     func discoverConfiguration(forUserId userId: String) {
         
         let hostname = StateHandler.hostForUser(userId: userId)
-        os_log("Discovering service config for '%{public}@'", log: .default, type: .debug, hostname)
+        os_log("Discovering service config for '%{public}@'...", log: .default, type: .debug, hostname)
         
         let discoveryURL = URL(string: "https://\(hostname)")!
+        
         OIDAuthorizationService.discoverConfiguration(forIssuer: discoveryURL) { (configuration, error) in
-            
-            if let error = error {
+            // 1001: Timeout, 1003 = Hostnot found
```

Comment:
```suggestion
            // 1001: Timeout, 1003 = Host not found
```

Reply:
One of my coding notes. Using system constants now.